e2e/pages/events/draftCMOEvent.page.js

```diff
@@ -0,0 +1,11 @@
+const { I } = inject();
+
+module.exports = {
+  fields: {
+    cmoHearingDateList: '#cmoHearingDateList',
+  },
+  draftCMO(date= '1 Jan 2050') {
```

Comment:
minor, but formatting could be better, to be honest not sure if that should be defaulted. 

The name doesn't seem to be correct we are selecting the date/hearing, not sure if it should be called draftCMO.

Reply:
@akomisarek I think in place of 'draftCMO', 'associateHearingDate' would be better.

Reply:
Why not just `selectHearingDate`?

Reply:
As our story title is "Associate new CMO to a specific hearing", I thought 'associateHearingDate' suits better.